drivers/android/process.rs

```diff
@@ -872,19 +888,17 @@ impl FileOperations for Process {
 
         // Do similar dance for the state lock.
         let mut inner = obj.inner.lock();
-        let mut threads = BTreeMap::new();
-        let mut nodes = BTreeMap::new();
-        swap(&mut inner.threads, &mut threads);
-        swap(&mut inner.nodes, &mut nodes);
+        let threads = replace(&mut inner.threads, RBTree::new());
```

Comment:
`core::mem::take`?

Reply:
I had `swap` elsewhere and @bjorn3 suggested `replace`. Now that I finally replaced all instances of `swap` I learn about `take`.

I'll make this change.